Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:0.10-onbuild
+
+VOLUME ["/project"]
+EXPOSE 6789
+
+CMD ["serve","/project/project.yml"]
```

Comment:
Not realy you can ovverrid the command and arguments:
`docker run -v /home/julien/my-super-project/:/project -p 6789 kosmtik/kosmtik:latest serve /project/youpla.yml`